### ISSUE TITLE
Alerts fire when email is read, not received

### ISSUE THREAD

**Issue by mclaborn** [tjeb/Mailbox-Alert#2]
Thunderbird 52.9.1 (64-bit) on Ubuntu 16.04

A bunch of emails will arrive in my inbox while the computer is shut down. When I open Thunderbird and go through the inbox reading those messages that arrived while TB was closed, the alerts will fire when the email is read/opened.  I think they are only supposed to fire when the email arrives? It is useless to fire an alert when I read an email because I am already reading it - I don't need an alert.

**Comment by tjeb** [comment]
hmm, no that is indeed not intended. Maybe the 'fire immediately' change from a while ago causes this (as in, it fires the alert before it realizes it shouldn't). Can you see if it is better when you set it to a (small) delay? Right-click on any folder, Mailbox Alert -> Delay Before Alert -> X seconds. It's a global setting so only need to set it for 1 folder.

**Comment by mclaborn** [comment]
I set the delay to 3 seconds on the folder where I was seeing it the most and have not seen it since.

**Comment by mclaborn** [comment]
With the delay at 3 seconds I still see this periodically, but not as frequently as before.

**Comment by mclaborn** [comment]
I've narrowed down the problem somewhat.  There is a github project that I subscribe to with my personal email account. I set up a rule at my email host to forward those emails to my work email account. Those are the only messages that demonstrate this problem, when reading them in my work email account in Thunderbird. The alert pops up when I delete one of those messages that I've read and it appears to be an alert for the message that I've just deleted. It does not happen every time.
